CardCollectionView/Source/Controller/Navigation Controller/NavigationViewController.swift

```diff
@@ -10,35 +10,35 @@ import UIKit
 
 class NavigationViewController: UINavigationController {
 
-    var statusBarStyle = UIStatusBarStyle.LightContent
+    var statusBarStyle = UIStatusBarStyle.lightContent
     
-    override func preferredStatusBarStyle() -> UIStatusBarStyle {
-        return statusBarStyle
-    }
+//    override func preferredStatusBarStyle() -> UIStatusBarStyle {
```

Comment:
`preferredStatusBarStyle` is now implemented as a property in Swift 3, this should be reflected instead of commenting out the functionality e.g.: ```override var preferredStatusBarStyle: UIStatusBarStyle {
        return .lightContent
    }```